lib/std/fmt.zig

```diff
@@ -491,6 +491,13 @@ pub fn formatType(
             return format(context, Errors, output, "{}@{x}", .{ @typeName(T), @ptrToInt(value) });
         },
         .Type => return output(context, @typeName(T)),
+        .EnumLiteral => {
+            const name = @tagName(value);
+            var buffer: [name.len + 1]u8 = undefined;
+            buffer[0] = '.';
+            std.mem.copy(u8, buffer[1..], name);
```

Comment:
If the enum name is not a valid identifier then you'll need to escape it.

Reply:
works just fine: `std.debug.warn("hello {}\n", .{.@"123"});`

| `hello .123`

Reply:
IMO that doesn't look great, it should render as `hello .@"123"`.
Messier is something like: `.@"some .fun enum?"`

Reply:
This PR has is at parity with other types in fmt -- no quoting. If implemented it should be consistent across all types.

```zig
    const @"123" = struct {};
    var foo = @"123"{};
    var p = &foo;
    const Bar = enum { one, @"2two" };

    std.debug.warn("hello {}\n", .{p});
    std.debug.warn("hello {}\n", .{Bar.@"2two"});
    std.debug.warn("hello {}\n", .{.@"123"});
```

```
hello 123{ }
hello .123
hello Bar.2two
```